static/js/common/upload-addon.js

```diff
@@ -341,7 +341,7 @@
                             ], current, matched, messages = [],
                             // this.id is in the form ["testcases_javascript_instanceactions", "_call_expression", "createelement_variable"],
                             // we usually only match one of the elements.
-                            matchId = function (id) {return _.contains(this.id, id);};
+                            matchId = function (id) {return (this.hasOwnProperty('id') && _.contains(this.id, id));};
```

Comment:
I don't think the parens are needed here. Might be nice to break the `return` onto its own line now that this is a bit longer too.